terraform/paas/terraform_qa.tfvars

```diff
@@ -0,0 +1,16 @@
+app = {
+  name         = "qa-publish-teacher-training-courses"
+  docker_image = "dfedigital/publish-teacher-training:latest"
```

Comment:
Are we deploying `latest` every time? How do we deploy specific versions?

Reply:
We could use the strategy     = "blue-green-v2" (line 5) in the same file to deploy different versions.